scripts/asm-installer/install_asm

```diff
@@ -2037,6 +2043,45 @@ register_gce_identity_provider() {
   retry 3 kubectl apply -f asm/identity-provider/googleidp.yaml
 }
 
+enable_service_mesh_feature() {
+  info "Enabling the service mesh feature..."
+
+  local TOKEN
+  TOKEN="$(retry 2 gcloud \
+    --project="${PROJECT_ID}" auth print-access-token)"
+
+  retry 2 run curl -s -H "Content-Type: application/json" \
+    -XPOST "https://gkehub.googleapis.com/v1alpha1/projects/${PROJECT_ID}/locations/global/features?feature_id=servicemesh"\
```

Comment:
Can you pass feature_id as part of the JSON payload instead? That would make it possible to use a prefix template for the rest of the URL instead of having to copy/paste it to two functions with a slight difference.

Reply:
Discussed in chat. Not feasible.